tokenserver-db-postgres/src/models.rs

```diff
@@ -0,0 +1,31 @@
+use std::time::Duration;
+
+use super::pool::Conn;
+use syncserver_common::Metrics;
+
+#[allow(dead_code)]
+pub struct TokenserverDb {
```

Comment:
we should probably call this `TokenserverPgDb` but plenty of time to change it later